src/quicknxs/interfaces/configuration.py

```diff
@@ -45,14 +45,14 @@ class Configuration(object):
     # Direct beam uses the same low res roi as the data run
     lock_direct_beam_y = False
     # Number of events below which we throw away a workspace
-    # Note: not exposed in the UI, but can be modified in ~/.config/.refredm.conf
+    # Note: not exposed in the UI, but can be modified in ~/.config/.quicknxs.conf
```

Comment:
I guess we can remove this note, since we don't read this parameter from the config file anymore